packages/topotal-ui/src/components/TextArea/index.tsx

```diff
@@ -49,6 +51,15 @@ export const TextArea = forwardRef(({
     onChangeText?.(text)
   }, [onChangeText])
 
+  const handleKeyPress = useCallback((event: NativeSyntheticEvent<TextInputKeyPressEventData>) => {
+    const { key } = event.nativeEvent
+    const keyEvents = event.nativeEvent as any
+
+    if((event.nativeEvent.key === 'Enter' && keyEvents.metaKey) || (key === 'Enter' && keyEvents.ctrlKey)) {
+      onCmdEnterPress?.()
+    }
+  }, [onCmdEnterPress])
+
```

Comment:
これだと TextArea を使う側が onCmdEnterPress は使わず onKeyPress で何かコマンドとエンターを取得したい場合に困るので（たぶんそれはまれ）こうした方がいいかも 👍🏻 TextInput も同様。
```suggestion
  const handleKeyPress = useCallback((event: NativeSyntheticEvent<TextInputKeyPressEventData>) => {
    const { key } = event.nativeEvent
    const keyEvents = event.nativeEvent as any

    if((event.nativeEvent.key === 'Enter' && keyEvents.metaKey) || (key === 'Enter' && keyEvents.ctrlKey)) {
      onCmdEnterPress?.()
      return
    }
    
    onKeyPress?.(event)
  }, [onCmdEnterPress])

```

Reply:
ありがとうございます！
完全理解しました 🙇 

こちら修正し、story上でも`onKeyPress`に影響ないことがわかるようなstoryを追加しました！